### DISCUSSION TITLE
B61-13 with F-35A

### DISCUSSION THREAD

**Discussion by NaldanoMalvo** [PygmalionOfCyprus/cmo-db-requests#5345] [Suggestions]
According to the Center for Arms Control and Non-Proliferation factsheet (https://armscontrolcenter.org/u-s-nonstrategic-nuclear-weapons/) 
"The B61-13 would be carried by the same bombers and fighters as the B61-12" 
Therefore B61-13 should be F-35A compatible by now and is no longer B-21 only.

**Comment by PygmalionOfCyprus** [comment]
This should be opened as an issue, not a discussion.
